node/tsconfig.json

```diff
@@ -25,6 +25,15 @@
         ] /* Specify a set of bundled library declaration files that describe the target runtime environment. */,
         "outDir": "./build-ts" /* Specify an output folder for all emitted files.*/
     },
+    "ts-node": {
+        "transpileOnly": true,
+        "compilerOptions": {
+            "module": "CommonJS",
+            "target": "ES2018",
+            "esModuleInterop": true
+        },
+        "esm": true
```

Comment:
Does Interloping have any detrimental performance implications?  
I'd be concerned about adding this to the project.  It's kind of a development feature.

Reply:
Will revert this file, added for testing